geometry/grassmann_body.hpp

```diff
@@ -236,7 +238,9 @@ inline Bivector<quantities::Product<LScalar, RScalar>, Frame> Commutator(
 template<typename Scalar, typename Frame, int rank>
 inline Multivector<double, Frame, rank> Normalize(
     Multivector<Scalar, Frame, rank> const& multivector) {
-  return multivector / multivector.Norm();
+  Scalar const norm = multivector.Norm();
+  CHECK_NE(Scalar(), norm);
+  return multivector / norm;
```

Comment:
Because I spent a lot of time tracking all the NaNs that would propagate to weird places.  You should be able to assume that the result actually has norm 1.  NaNs are not useful in this context as they break the postcondition.
For instance, there is a benchmark where this check fires.  It's better to know about it than to silently benchmark how the processor handles NaNs.